R/nlmixr2output.R

```diff
@@ -163,84 +148,70 @@
   )
 }
 
-#'  This will add the between subject variability to the mu-referenced theta.  It also expands the table to include non-mu referenced ETAs
-#'
+#'  This will add the between subject variability to the mu-referenced theta.
+#'  It also expands the table to include non-mu referenced ETAs
 #'
 #' @param .ret The focei return environment
 #' @param .ui The rxode2 ui environment
-#' @return Nothing called for side effects on popDf and popDfSig in the .ret environment
+#' @returns Nothing called for side effects on popDf in the .ret environment
 #' @author Matthew L. Fidler
 #' @noRd
-.updateParFixedAddBsv <- function(.ret, .ui) {
-  .ome <- .ret$omega
-  .omegaFix <- as.data.frame(.ui$ini)
+.updateParFixedAddBsv <- function(popDf, iniDf, omega, .sigdig, .muRefDataFrame, .muRefCurEval) {
+  .omegaFix <- as.data.frame(iniDf)
   .omegaFix <- .omegaFix[is.na(.omegaFix$ntheta), ]
   .omegaFix <- setNames(.omegaFix$fix, paste(.omegaFix$name))
-  .sigdig <- rxode2::rxGetControl(.ui, "sigdig", 3L)
```

Comment:
This drops the use of `sigdig` in the output.  In theory it should match the estimation method if specified.